modules/light-clients/08-wasm/keeper/grpc_query.go

```diff
@@ -46,7 +46,10 @@ func (k Keeper) CodeHashes(goCtx context.Context, req *types.QueryCodeHashesRequ
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
 	var codeHashes []string
-	storedHashes := types.GetCodeHashes(ctx, k.cdc)
+	storedHashes, err := types.GetCodeHashes(ctx, k.cdc)
+	if err != nil {
+		return nil, status.Error(codes.DataLoss, err.Error())
+	}
 	for _, hash := range storedHashes.Hashes {
```

Comment:
CR for breathing room maybe? :) 

```suggestion
	}
	
	for _, hash := range storedHashes.Hashes {
```